qiita_db/software.py

```diff
@@ -455,6 +482,38 @@ def add_publications(self, publications):
             qdb.sql_connection.TRN.add(sql, sql_params, many=True)
             qdb.sql_connection.TRN.execute()
 
+    @property
+    def environment_script(self):
+        """The script used to start the plugin environment
+
+        Returns
+        -------
+        str
+            The script used to start the environment
```

Comment:
Path to the scrip - not necessarily full as it will be executed inside the environment in which the plugin lives, so it should be added in the path already.